### PR TITLE
[TCD-1749] Added cicd doc for api-testing mark3

### PR DESCRIPTION
**Pull request by spider-sauce**
### Description

Added new CICD process doc for API Testing mark3

### Motivation and Context

- Outline new CICD tool
- Demonstrate how to generate required WebHook in the dashboard

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD) document.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/api-testing/mark3/quick-start/integrate-with-your-cicd.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    
+## Commands
+
+### `run`
+
+__Description__: Runs a single test. The `{hookId}` and `testId` are both required.
+
+__Example__:
+```bash
+docker run quay.io/saucelabs/apifctl run -H https://foobar:ae5499d58ab1@api.us-west-1.saucelabs.com/api-testing/rest/v4/4be5-45db-a9aa-4c5d9027e6f5 -i 555161be0ad1b20c6e2dd020
```
Comment (deejay-sl) [inline docs/api-testing/mark3/quick-start/integrate-with-your-cicd.md:59]: :exclamation:**Cycode**: Secret of type: 'Username And Password In Uri' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |

File: docs/api-testing/mark3/quick-start/integrate-with-your-cicd.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+```
+---
+
+### `run-all`
+
+__Description__: Runs all the complete tests in the project referenced by the hook.
+
+__Example__:
+```bash
+docker run quay.io/saucelabs/apifctl run-all -H https://foobar:ae5499d58ab1@api.us-west-1.saucelabs.com/api-testing/rest/v4/4be5-45db-a9aa-4c5d9027e6f5
```
Comment (deejay-sl) [inline docs/api-testing/mark3/quick-start/integrate-with-your-cicd.md:69]: :exclamation:**Cycode**: Secret of type: 'Username And Password In Uri' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |

File: docs/api-testing/mark3/quick-start/integrate-with-your-cicd.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+```
+---
+
+### `run-tag`
+
+__Description__: Runs all the tests tagged with a specific tag.
+
+__Example__:
+```bash
+docker run quay.io/saucelabs/apifctl run-tag -H https://foobar:ae5499d58ab1@api.us-west-1.saucelabs.com/api-testing/rest/v4/4be5-45db-a9aa-4c5d9027e6f5 -tag production
```
Comment (deejay-sl) [inline docs/api-testing/mark3/quick-start/integrate-with-your-cicd.md:79]: :exclamation:**Cycode**: Secret of type: 'Username And Password In Uri' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |

File: docs/api-testing/mark3/quick-start/integrate-with-your-cicd.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+```
+---
+
+### `exec`
+
+__Description__: Sends a test residing in the local file system (directory with `unit.xml` and `input.xml`) to the cloud for execution.
+
+__Example__:
+```bash
+docker run -v $(pwd)/tests:/tests quay.io/saucelabs/apifctl exec -H https://foobar:ae5499d58ab1@api.us-west-1.saucelabs.com/api-testing/rest/v4/4be5-45db-a9aa-4c5d9027e6f5 -p /tests/test_abc
```
Comment (deejay-sl) [inline docs/api-testing/mark3/quick-start/integrate-with-your-cicd.md:89]: :exclamation:**Cycode**: Secret of type: 'Username And Password In Uri' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |

File: docs/api-testing/mark3/quick-start/integrate-with-your-cicd.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+```
+---
+
+### `upload`
+
+__Description__: Uploads a test residing in the local file system  (directory with `unit.xml` and `input.xml`) to the cloud for storage.
+
+__Example__:
+```bash
+docker run -v $(pwd)/tests:/tests quay.io/saucelabs/apifctl upload -H https://foobar:ae5499d58ab1@api.us-west-1.saucelabs.com/api-testing/rest/v4/4be5-45db-a9aa-4c5d9027e6f5 -p /tests/test_abc
```
Comment (deejay-sl) [inline docs/api-testing/mark3/quick-start/integrate-with-your-cicd.md:99]: :exclamation:**Cycode**: Secret of type: 'Username And Password In Uri' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |

File: docs/api-testing/mark3/quick-start/integrate-with-your-cicd.md (hunk 1 above)
Comment (theirish81) [reply docs/api-testing/mark3/quick-start/integrate-with-your-cicd.md:59]: #cycode_secret_ignore_everywhere
--


<br class="Apple-interchange-newline">

Comment (deejay-sl) [reply docs/api-testing/mark3/quick-start/integrate-with-your-cicd.md:59]: :+1: Cycode has finished processing your request. Please review your status checks.

**Pull request merged**